apps/mobile/src/common/global-sheet-provider.tsx

```diff
@@ -0,0 +1,29 @@
+import { createContext, useContext, useRef } from 'react';
```

Comment:
Do you have any thoughts on getting rid of the `common` folder? 

Maybe we can have the `providers` at root level and move the other items elsewhere?

We have `common/` and `shared/` and also another folder `filesystem/`.

Reply:
I actually moved some stuff out of this one with intention to rename it to something like `core`. What do you think?

Reply:
Perfect! I'm happy to have one folder for all the common/shared etc.